pkg/cluster/audit/audit.go

```diff
@@ -81,7 +82,7 @@ func ShowAuditList(dir string) error {
 
 // OutputAuditLog outputs audit log.
 func OutputAuditLog(dir string, data []byte) error {
-	fname := filepath.Join(dir, base52.Encode(time.Now().UnixNano()))
+	fname := filepath.Join(dir, base52.Encode(time.Now().UnixNano()+rand.Int63n(1000)))
```

Comment:
Better split this to another PR

Reply:
https://github.com/pingcap/tiup/pull/916